streams/src/main/java/org/apache/kafka/streams/kstream/internals/CogroupedStreamAggregateBuilder.java

```diff
@@ -201,9 +236,10 @@ private void processRepartitions(final Map<KGroupedStreamImpl<K, ?>, Aggregator<
             "-cogroup-merge",
             builder,
             CogroupedKStreamImpl.MERGE_NAME);
-        final ProcessorSupplier<K, VOut> passThrough = new PassThrough<>();
+        final KTableProcessorSupplier<K, VOut, VOut> passThrough = new KTablePassThrough<>(parentProcessors, queryableName);
```

Comment:
I noticed that `queryableName` is a different parameter than the one we actually build the parent processors with (`storeBuilder.name()`). It wouldn't surprise me if there's a subtle difference between them.

Reply:
Looks like they're the same when cogroup is materialized, but if it's not the `storeBuilder.name()` is auto generated and `queryableName` is null. With that said, it seems sto me that it would be better to pass `storeBuilder.name()` to the `passThrough`, to be safe. WDYT?

Reply:
Yes, that sounds like the right thing to do. Thanks!